irksome/stage.py

```diff
@@ -271,44 +260,26 @@ def getFormStage(F, butch, u0, t, dt, bcs=None, splitting=None, vandermonde=None
     for bc in bcs:
         bcarg = as_ufl(bc._original_arg)
 
-        gblah_cur = []
-
         if bc in bc_constraints:
             bcparams, bclower, bcupper = bc_constraints[bc]
-            for i in range(num_stages):
-                Vsp, Vbigi = stage2spaces4bc(bc, V, Vbig, i)
-                gdat = Function(Vsp)
-                vbc = TestFunction(Vsp)
-                gmethod = lambda gd, gc: solve(inner(gd - gc, vbc) * dx == 0,
-                                               gd, solver_parameters=bcparams)
             gcur = replace(bcarg, {t: t+C[i] * dt})
-            gblah_cur.append((gdat, gcur - Vander_col[i] * bcarg, gmethod))
+            gcur = gcur - Vander_col[i] * bcarg
```

Comment:
Is there a reason to prefer this over `gcur -= ...`?

Reply:
Constants cannot be updated

Reply:
Ah, I didn't think about that case.